### PR TITLE
Alfredapi 386 grad5

### PR DESCRIPTION
**Pull request by WimCrols**
Fixes https://xenitsupport.jira.com/browse/ALFREDAPI-386

- [x] Is [CHANGELOG.md](https://github.com/xenit-eu/alfred-api/blob/master/CHANGELOG.md) extended?
- [x] Does this PR avoid breaking the API? 
    Breaking changes include adding, changing or removing endpoints and/or JSON objects used in requests and responses.
- [x] Does the PR comply to REST HTTP result codes policy outlined in the [developer guide](https://github.com/xenit-eu/alfred-api/blob/master/developer-documentation)?
- [x] Does the PR follow our [coding styleguide and other active procedures](https://xenitsupport.jira.com/wiki/spaces/XEN/pages/624558081/XeniT+Enhancement+Proposals+XEP)?
- [x] Is usage of `this.` prefix avoided?

See [README.md](./README.md) for full explanation.

### CODE REVIEW

File: apix-docker/build.gradle [hunk 1]
```diff
@@ -16,9 +20,17 @@ subprojects {
...
     dockerCompose {
-        useComposeFiles = ["docker-compose.yml"] +
-                (JenkinsUtil.buildId == null ? ["debug-extension.docker-compose.yml"] : [])
+        useComposeFiles = ['docker-compose.yml']
+        // Don't use the dev compose file on Jenkins
+        if(JenkinsUtil.buildId == null) {
+            // Allow the dev compose file to be deleted
+            def extraComposeFile = 'debug-extension.docker-compose.yml'
+            if(project.file(extraComposeFile).exists()) {
+                useComposeFiles.add(extraComposeFile)
```
Comment (RVanhuysseXenit) [inline apix-docker/build.gradle]: Where does the debug compose file cause issues? If this causes issues on Jenkings, doesnt this also cause issues in local builds?

File: build.gradle [hunk 2]
```diff
@@ -44,25 +46,12 @@ buildscript {
...
-        }
-        mavenCentral()
-    }
-    dependencies {
-        classpath 'com.moowork.gradle:gradle-node-plugin:1.2.0'
-        classpath 'org.apache.httpcomponents:httpclient:4.5.3'
-        classpath 'org.apache.httpcomponents:httpmime:4.5.3'
-        classpath 'org._10ne.gradle:rest-gradle-plugin:0.4.2'
-        classpath 'org.codehaus.groovy.modules.http-builder:http-builder:0.7'
-        classpath 'eu.xenit.gradle:alfresco-docker-plugin:4.0.2'
```
Comment (RVanhuysseXenit) [inline build.gradle]: I suppose those buildscript dependencies are now included in the plugins?

File: apix-docker/build.gradle (hunk 1 above)
Comment (WimCrols) [reply apix-docker/build.gradle]: Locally, when having multiple version (51 and 60 for exmaple) running as containers, the debug port clashes.

File: build.gradle (hunk 2 above)
Comment (WimCrols) [reply build.gradle]: I have no idea, except that I tested it and everything works

**Pull request merged**